### DISCUSSION TITLE
I can't set the sliders amount per view. I made it per custom js and directly to the markup, but it didn't work.

### DISCUSSION THREAD

**Discussion by wdmarketLV** [breezefront/community#82] [1. Help]
I can't set the sliders amount per view. I made it per custom js and directly to the markup, but it didn't work.
It is slick slider - https://prnt.sc/2B6zymiz5rWh
By default for me showing 5 items on single product page, but here 1 is just for testing on screen.

I am using - $(document).on('click', '#kanmo-mc-search-btn', () => {})****

**Comment by vovayatsyuk** [comment]
We do not support these [options](https://breezefront.com/docs/slider#options). You need to use CSS to change the number of visible items.
